addons/l10n_in_edi/views/account_journal_views.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_account_journal_form_inherited" model="ir.ui.view">
+        <field name="name">account.journal.form.inherited</field>
+        <field name="model">account.journal</field>
+        <field name="inherit_id" ref="account.view_account_journal_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='group_edi_config']" position="attributes">
+                <attribute name="invisible">not l10n_in_edi_feature_enabled</attribute>
```

Comment:
Don't do this like this.
instead do it with `_is_compatible_with_journal`
this auto-hide it.

Reply:
done